test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be ignored and not appear in your remote repo. You can take a look in [this link](https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore) to help you how to remove files from the remote repo after push.